frame/staking/src/pallet/impls.rs

```diff
@@ -754,24 +779,17 @@ impl<T: Config> frame_election_provider_support::ElectionDataProvider<T::Account
 			return Err("Voter snapshot too big")
 		}
 
-		let slashing_span_count = <SlashingSpans<T>>::iter().count();
-		let weight = T::WeightInfo::get_npos_voters(
-			nominator_count,
-			validator_count,
-			slashing_span_count as u32,
-		);
-		Ok((Self::get_npos_voters(), weight))
+		Ok(Self::get_npos_voters())
```

Comment:
`get_npos_voters` accounts for its own weight, but seems like the reads for `CounterFor{Validators, Nominators}` here is unaccounted for?

Reply:
excellent point.